microsoft/runbook/azure.yml

```diff
@@ -3,11 +3,17 @@ parent:
   - path: ./tiers/tier.yml
 variable:
   - name: location
-    value: westus2
+    value: ""
   - name: reserve_environment
     value: "no"
+  - name: resource_group_name
+    value: ""
   - name: marketplace_image
     value: ""
+  - name: deploy
+    value: true
+  - name: wait_delete
```

Comment:
What is wait_delete for? how is it different from reserve_environement?

Reply:
The `wait_delete` means when an environment is deleting, whether to wait the deleting operation, or return immediately. It's used in some deployment, that the resource is limited. If the deletion is not completed, there may be errors when creating new environment. For example, reaching quota in Azure, memory bound in Hyper-V, and so on.
The `reserve_environment` means after test cases completed, keep this environment or not. It's used for deploy an environment for future uses, or troubleshooting.

@LiliDeng could you refine above information, and update to the runbook reference?

Reply:
BTW, you always explain reserve to mean keep. You see that keep is much more of a common/easy word for that, right? Time to rename?

Reply:
Sure, I will update it. This field is not used in production.

Reply:
@squirrelsc Seems you already change for keep_environment?

Reply:
yes, it's in this PR.